core/control-plane/contract-core/src/test/java/org/eclipse/edc/connector/contract/negotiation/command/handlers/DeclineNegotiationCommandHandlerTest.java

```diff
@@ -49,7 +49,6 @@ void handle_negotiationExists_declineNegotiation() {
                 .protocol("test-protocol")
                 .state(ContractNegotiationStates.REQUESTED.code())
                 .build();
-        var originalTime = negotiation.getUpdatedAt();
         var command = new DeclineNegotiationCommand(negotiationId);
```

Comment:
why was this deleted? I don't agree with removing test assertions unless there is a good reason.

Reply:
Yes, I was just annoyed by this test and thought that behavior was already tested on the `ContractNegotiation`, but in fact it's not, anyway I made it not flaky anymore.